jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -62,6 +62,10 @@
             branch: 'release-1.3'
             timeout: 50
             job-env: ''  # Empty expected
+        - 'build-1.4':
+            branch: 'master' # TODO: Change this to release-1.4 when it is cut
```

Comment:
Will please note that when we move this to branch: 'release-1.4' we should also delete all the 1.2 jobs at this time?